addon/components/paper-virtual-repeat.js

```diff
@@ -60,14 +60,28 @@ export default VirtualEachComponent.extend({
     if (!this.get('itemHeight')) {
       let elem = this.get('containerSelector') ? $(this.get('containerSelector'))[0].firstElementChild : this.$('.md-virtual-repeat-offsetter')[0].firstElementChild;
       if (elem) {
-        this.set('itemHeight', this.get('horizontal') ? elem.offsetWidth : elem.offsetHeight);
-        this.set('_totalHeight', Math.max((this.get('length') ? this.get('length') : get(this.get('items'), 'length')) * this.get('itemHeight'), 0));
+        run.schedule('actions', () => {
+          this.set('itemHeight', this.get('horizontal') ? elem.offsetWidth : elem.offsetHeight);
+          this.set('_totalHeight', Math.max((this.get('length') ? this.get('length') : get(this.get('items'), 'length')) * this.get('itemHeight'), 0));
+        });
         // this.rerender();
       }
     }
-    this.set('height', this.get('horizontal') ? this.$()[0].clientWidth : this.$()[0].clientHeight);
+    run.schedule('actions', () => {
+      this.set('height', this.get('horizontal') ? this.$()[0].clientWidth : this.$()[0].clientHeight);
+    });
 
   },
+  offsetterStyle: computed('horizontal', '_marginTop', {
+    get() {
```

Comment:
We're using `computed('key', function() {` style for get computeds in the project.